src/actions.js

```diff
@@ -1,8 +1,8 @@
 export default {
   BUILD: 'build',
   DEVELOP: 'develop',
-  INSTALL: 'install',
-  LINT: 'lint',
```

Comment:
I've started to namespace these just like we do in the `package.json` so that we can implement a `sagui test` soon that runs all of these actions.